build-utils/npm-export/index.d.ts

```diff
@@ -1,2 +1,3 @@
 export * from "./types/main"
-export * from "./types/typing/index"
+import * as harFormat from "har-format";
+export { harFormat }
```

Comment:
re-export HAR typings from npm module via the `harFormat` namespace

<img width="654" alt="screen shot 2017-03-14 at 10 53 58 pm" src="https://cloud.githubusercontent.com/assets/1680390/23904294/ab85d8c2-090a-11e7-841e-956c8e892556.png">